spec/erroneous_win_bug_spec.rb

```diff
@@ -0,0 +1,116 @@
+require './lib/board'
+require './lib/player'
+require './lib/turn'
+require './lib/game'
+
+RSpec.describe Board do
+  let(:game) {Game.new}
+  
+  it "doesn't have an erroneous win conidtion" do
+    game.board.game_board['A'].push('X')
```

Comment:
would the "letter".next method be a good option for refactoring to check the board? (within the limits of the keys?